sn_node/src/replication.rs

```diff
@@ -163,7 +163,7 @@ impl Node {
                         "Can not fetch record {pretty_key:?} from node {holder:?}, fetching from the network"
                     );
                     node.network
-                        .get_record_from_network(key, None, false)
+                        .get_record_from_network(key, None, GetQuorum::All, false)
```

Comment:
this qill requires CLOSE_GROUP_SIZE copies to be fetched during replication, which in most cases when a churn happens won't got satisfied.
Consider change this to `majority` ?